docs/configuration/git-providers/create-github-app.md

```diff
@@ -0,0 +1,47 @@
+# Create and configure a new GitHub App for GitHub Enterprise
```

Comment:
We need the documentation for creating a github app for GitHub Cloud

Reply:
@pauloribeiro-codacy this page should be mostly the same. any thoughts?

Reply:
Let's merge this PR now, and I'll continue working on updating the instructions for the Git providers on this other PR: https://github.com/codacy/chart/pull/350.

I'll figure out a way of sharing this information between GitHub Cloud and Enterprise. :+1: